### ISSUE TITLE
Frontend Doesn't work

### ISSUE THREAD

**Issue by kaziqta** [chrisleekr/binance-trading-bot#35]
The frontend throws the following error in the console
`
Uncaught Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at fd (react-dom.production.min.js:185)
    at n (react-dom.production.min.js:87)
    at w (react-dom.production.min.js:90)
    at react-dom.production.min.js:94
    at U (react-dom.production.min.js:110)
    at Pj (react-dom.production.min.js:234)
    at di (react-dom.production.min.js:168)
    at Nj (react-dom.production.min.js:168)
    at sc (react-dom.production.min.js:168)
    at gf (react-dom.production.min.js:162)
`

**Comment by chrisleekr** [comment]
As that is not development env, it is not possible to find what causes that.

Try clean up all volumes and re-launch. You will lose the configuration and last buy price records. So remember that in somewhere.

```
$ docker-compose -f docker-compose.server.yml down -v
$ docker-compose -f docker-compose.server.yml up -d
```

**Comment by kaziqta** [comment]
Yup, recreating it all day long, pruning the engine etc, same thing.

**Comment by 4KJuegos** [comment]
Yep, doesn't work. It try to refresh, but nothing.

**Comment by chrisleekr** [comment]
Ah, I got the same error now. Something must happen with React.js CDN. Let me check.

**Comment by kaziqta** [comment]
> Ah, I got the same error now. Something must happen with React.js CDN. Let me check.

I got exactly the same impression, the issue is not with the bot itself.

**Comment by chrisleekr** [comment]
The issue must be resolved with v0.0.56

https://github.com/chrisleekr/binance-trading-bot/releases/tag/v0.0.56
